### PR TITLE
Create SN-Copilot integration

### PR DESCRIPTION
**Pull request by Rampriya-S**
As a product manager preparing for Microsoft Copilot integration with ServiceNow, I need to set up Agile Scrum for my team. The document should cover roles, product vision, themes, epics, backlog prioritization, and sprint planning. It must ensure clear priorities and readiness for a seamless integration process.

### CODE REVIEW

**Comment by niamccash** [comment]
HI @Rampriya-S 
Thank you for your contribution to ServiceNow's Hacktoberfest event! Your PR looks good. Can I suggest that you add a more descriptive file name that begins with a verb. Example: "Create a document for ... "

**Comment by earlduque** [comment]
Hello! This comment is to inform you that unfortunately, this repository, on October 3, was excluded from participation in the official Hacktoberfest event. As I investigate into why it was excluded, I would like to encourage you to participate in our other Hacktoberfest projects so that you still get completion progress for the official Hacktoberfest event.

We will count this contribution (if accepted and merged) for progress made for ServiceNow rewards (make sure to check out the other eligibility requirements in the blog post), but if you want the rewards from the official event, you will need to make sure you have at least 4 Pull Requests merged into _other_ projects by the end of October. We apologize for the inconvenience and appreciate your participation regardless!

Earl

File: SN-Copilot integration [hunk 1]
```diff
@@ -0,0 +1 @@
+As a product manager preparing for Microsoft Copilot integration with ServiceNow, I need to set up Agile Scrum for my team. The document should cover roles, product vision, themes, epics, backlog prioritization, and sprint planning. It must ensure clear priorities and readiness for a seamless integration process.
```
Comment (SapphicFire) [inline SN-Copilot integration:1]: Please consider aligning this prompt to the RISEN framework. I've checked off the components that are currently reflected in this prompt:

- [ ] R — Role: Define the AI's role. (e.g., "Act as an agile coach.")
- [x] I — Instructions: Provide clear instructions. (e.g., "Write a guide on crafting a good agile development story.")
- [x] S — Steps: Outline specific steps. (e.g., "Start with a user persona, describe the user's goal, list the tasks to achieve the goal, and define the acceptance criteria.")
  - Steps are not very clear
- [ ] E — End goal: State the goal and audience. (e.g., "Help team members understand how to write clear and effective agile stories.")
- [ ] N — Narrowing: Add constraints. (e.g., "Max 800 words, avoid technical jargon, use practical examples.")

**Review by SapphicFire — CHANGES_REQUESTED** [review]
Thank you for your submission. However, this pull request requires changes before it can be accepted. It is not placed in the `Prompts` directory, and does not provide a rich enough prompt for the AI to generate repeatable and quality results. Please complete your contribution by adding the necessary elements and resubmit or update this pull request for further review. Closing this for now. Once you make additional changes, feel free to re-open this Pull Request or create a new one.

**Pull request closed without merging**